sdk/iOS/src/WindowsAzureMobileServices.h

```diff
@@ -19,6 +19,32 @@
 #import "MSCoreDataStore.h"
 #import "MSPush.h"
 #import "MSDateOffset.h"
+#import "MSQueryResult.h"
```

Comment:
When building against the Mac framework target, it gave me a warning about no includes for public header files. These were the full list of public headers. Reviewing now, this list is probably smaller. I can review the public headers from the iOS static library and update.